vortex-filtering/include/vortex_filtering/filters/pdaf.hpp

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <Eigen/Dense>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+#include <vortex_filtering/vortex_filtering.hpp>
+
+namespace vortex::filter {
+
+template <concepts::DynamicModelLTVWithDefinedSizes DynModT, concepts::SensorModelLTVWithDefinedSizes SensModT> class PDAF {
+public:
+  static constexpr int N_DIM_x = DynModT::N_DIM_x;
+  static constexpr int N_DIM_z = SensModT::N_DIM_z;
+  static constexpr int N_DIM_u = DynModT::N_DIM_u;
+  static constexpr int N_DIM_v = DynModT::N_DIM_v;
+  static constexpr int N_DIM_w = SensModT::N_DIM_w;
+
+  using T = Types_xzuvw<N_DIM_x, N_DIM_z, N_DIM_u, N_DIM_v, N_DIM_w>;
+
+  using EKF            = vortex::filter::EKF_t<N_DIM_x, N_DIM_z, N_DIM_u, N_DIM_v, N_DIM_w>;
+  using Gauss_z        = typename T::Gauss_z;
+  using Gauss_x        = typename T::Gauss_x;
+  using Vec_z          = typename T::Vec_z;
+  using MeasurementsZd = std::vector<Vec_z>;
+  using StatesXd       = std::vector<Gauss_x>;
+  using GaussMixZd     = vortex::prob::GaussianMixture<N_DIM_x>;
```

Comment:
Wrong dimension. Tests should catch this

```suggestion
  using GaussMix_z     = vortex::prob::GaussianMixture<N_DIM_z>;
```

Reply:
Yeah my bad. But the naming was wrong not the actual type. So, N_DIM_x is right.